app/views/proposals/_proposal.html.erb

```diff
@@ -71,23 +71,23 @@
                       <div class="mb-3 row">
                         <h4>Subject Areas</h4>
                         <div class="col-6 mt-2">
-                          <label>Subject Area</label>
+                          <label class="required">Subject Area</label>
                           <%= f.select :subject_id, options_for_select(subjects_area, @proposal.subject&.id), { include_blank: '' }, disabled: action, class: "form-select"%>
                           <% if params[:publish] && @proposal.subject&.id.nil? %>
                             <p style="color: red">Subject is required</p>
                           <% end %>
                         </div>
                         <div class="col-6 mt-2">
                           <div class="mb-3">
-                            <label>AMS Subject Code 1</label>
+                            <label class="required">AMS Subject Code 1</label>
                             <%= f.select :ams_subjects, options_for_select(ams_subjects_code, proposal_ams_subjects_code(@proposal, 'code1')), { include_blank: '' }, disabled: action, class: "form-select", name:'ams_subjects[code1]'%>
                             <% if params[:publish] && proposal_ams_subjects_code(@proposal, 'code1').blank? %>
                               <p style="color: red">AMS Subject Code 1 is required</p>
                             <% end %>
                           </div>
 
                           <div class="mb-3">
-                            <label>AMS Subject Code 2</label>
+                            <label class="required">AMS Subject Code 2</label>
                             <%= f.select :ams_subjects, options_for_select(ams_subjects_code, proposal_ams_subjects_code(@proposal, 'code2')), { include_blank: '' }, disabled: action, class: "form-select", name:'ams_subjects[code2]'%>
                             <% if params[:publish] && proposal_ams_subjects_code(@proposal, 'code2').blank? %>
                               <p style="color: red">AMS Subject Code 2 is required</p>
```

Comment:
Would it make sense to use the same "required" class for the proposal_fields/_partials ?